src/jdk.compiler/share/classes/com/sun/source/doctree/EscapeTree.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.sun.source.doctree;
+
+import javax.lang.model.element.Element;
+import javax.lang.model.util.Elements;
+
+/**
+ * A tree node for a character represented by an escape sequence.
+ *
+ * <p>The following escape sequences are supported:
+ * <ul>
+ *     <li>{@code @@}, representing {@code @}, where it would otherwise be treated as introducing a block or inline tag,
+ *     <li>{@code @/}, representing {@code /}, as part of {@code *@/} to represent <code>&ast;&sol;</code>, and
```

Comment:
For other reviewers. Interpret that middle bullet literally: `@` escapes `/` only if it is preceded by `*`. In other words, it's a triple `*@/`, not a pair `@/`.

`@/` without preceding `*` causes errors if it can be confused for a start of tag.

For a block tag:
```
    error: no tag name after '@'
         @/
          ^
```
For an inline tag:
```
    error: no tag name after '@'
         {@/
          ^
```
(FWIW, @jonathan-gibbons, note that the caret positions differ: the former points to `/` while the latter points to `@`. )

`*` is a sort of looked-behind and not consumed, which can be seen from the fact that
```
    /**
     @*@/
     */
```
is translated to this in documentation
```
    */
```
Given all of the above, I'm not sure if `@/` can be technically qualified as an ordinary escape sequence.

Reply:
'/' cannot be part of a tag name.

A tag name starts with a character for which `isUnicodeIdentifierStart` is true, followed by zero or more characters for which `isUnicodeIdentifierPart` is true or is one of `.`, `-`, `:`.  See `DocCommentParser.readTagName` for verification. (We should update the Doc Comment Spec for that as well.)

So, the text of error messages above are correct, even if the positions require tweaking.

Reply:
Note that all the proposed escape sequences are context-sensitive, and are only enabled in appropriate circumstances.

Reply:
updated code, comments and spec.

Reply:
> (FWIW, @jonathan-gibbons, note that the caret positions differ: the former points to / while the latter points to @. )

I see the problem and think I understand the fix, but it is unrelated to the work here.  I'll defer this to a separate JBS issue and PR.